lib/strong_resources/controller/mixin.rb

```diff
@@ -21,10 +22,24 @@ def update_action?
         action_name == 'update'
       end
 
+      private
+
+      def _params_type
```

Comment:
I would name this something a little more explicit, like `jsonapi_type`. Otherwise I am thinking 'type of parameters' or type coercion.

Also, Could we move this to an internal class so the controllers themselves aren't polluted?